lib/themes/dosomething/paraneue_dosomething/scss/patterns/_counter.scss

```diff
@@ -10,6 +10,18 @@
   }
 }
 
+.card__column {
+  .photo__actions, .caption {
```

Comment:
The `.photo__action` is a [BEM element](http://getbem.com/introduction/) which indicates that it's semantically tied to the `.photo` block. If we start using that class elsewhere (without explicitly indicating that it's a standalone component), we make our styles more fragile to change.